paddle/framework/net.h

```diff
@@ -103,18 +79,10 @@ class Net {
 class PlainNet : public Net {
```

Comment:
I think net will have a common interface `AddOp`, so I add the net base interface with only AddOp